### PR TITLE
Pi 2131

### PR DESCRIPTION
**Pull request by izzyL-DJ**

### CODE REVIEW

File: server/views/pages/personal-details.njk [hunk 1]
```diff
@@ -238,6 +238,10 @@
             <p>
             <a href="/case/{{ crn }}/handoff/oasys">View sentence plan in OASys</a>
             </p>
+
+            <p>
+            <a href="{{ sentencePlan }}">View sentence plan in UI</a>
```
Comment (pmcphee77) [inline server/views/pages/personal-details.njk:243]: `sentencePlan` doesnt resolve to anything here. We need to add a ts function like this in `server/utils/utils.ts`

```
export const sentencePlanLink = () => {
  return `${config.sentencePlan.link}`
}
```
then add the function to the global nunJucks in `server/utils/nunJucksSetup.ts`

`njkEnv.addGlobal('sentencePlanLink', sentencePlanLink())`

Then the function can be called from the nunjucks template

`<a href="{{ sentencePlanLink }}" target="_blank">View sentence plan in UI</a>`

Also i think this should open in a new window hence i have specified `target="_blank"` above in the href

**Review by pmcphee77 — CHANGES_REQUESTED** [review]

File: server/views/pages/personal-details.njk [hunk 2]
```diff
@@ -238,6 +238,10 @@
             <p>
             <a href="/case/{{ crn }}/handoff/oasys">View sentence plan in OASys</a>
             </p>
+
+            <p>
+            <a href="{{ sentencePlanLink }}" target="_blank">View sentence plan in UI</a>
```
Comment (pmcphee77) [inline server/views/pages/personal-details.njk:243]: should be sentencePlanLink()

**Review by pmcphee77 — CHANGES_REQUESTED** [review]

**Review by pmcphee77 — DISMISSED** [review]

File: server/views/pages/personal-details.njk [hunk 3]
```diff
@@ -238,6 +238,10 @@
             <p>
             <a href="/case/{{ crn }}/handoff/oasys">View sentence plan in OASys</a>
             </p>
+
+            <p>
+            <a href="{{ sentencePlanLink() }}" target="_blank">View sentence plan in UI</a>
```
Comment (marcus-bcl) [inline server/views/pages/personal-details.njk:243]: Should this say something other than "UI"?  Maybe something like "View in the sentence plan service"? I'm not sure....

**Review by marcus-bcl — APPROVED** [review]
Looks good

**Review by pmcphee77 — APPROVED** [review]

**Pull request merged**